### PR TITLE
BUGFIX: Use correct Icons for FA5

### PR DESCRIPTION
**Pull request by mstruebing**
Resolves: #47

### CODE REVIEW

File: Configuration/NodeTypes.yaml [hunk 1]
```diff
@@ -19,11 +19,11 @@
         stats:
           label: 'Neos.GoogleAnalytics:NodeTypes.StatsTabMixin:tabs.stats'
           position: 100
-          icon: 'icon-line-chart icon-chart-line'
+          icon: 'chart-line'
```
Comment (aertmann) [inline Configuration/NodeTypes.yaml]: this breaks backwards compatibility.. just add chart-line in addition to the other ones so we don't need a new version for the latest version

Comment (mstruebing) [reply Configuration/NodeTypes.yaml]: Hmm, wenn I simply add '`chart-line` to the other ones no Icon is shown at least in the new UI:
`icon: 'icon-line-chart icon-chart-line chart-line'`

Comment (aertmann) [reply Configuration/NodeTypes.yaml]: did you try adding it as the first instead? if that doesn't work I guess they're changed how the selector works as well

**Comment by kdambekalns** [comment]
Cannot reproduce. Feel free to reopen if this is still an issue with current package versions.

**Pull request closed without merging**